### ISSUE TITLE
What guide should I use to install Homebridge in Raspberry?

### ISSUE THREAD

**Issue by Ricardo1980** [nfarina/homebridge#660]
I see there are 2 guides to install homebridge in my raspberry pi.

https://github.com/nfarina/homebridge/wiki/Running-HomeBridge-on-a-Raspberry-Pi
(this installs node 4.0.0)

https://github.com/nfarina/homebridge/issues/103
(this installs node 0.12 and it seems homebridge is inside my home folder)

The thing is that I cannot make my plugins work with dependencies and I think it is because I messed up everything.

What is the proper way to install homebridge in raspberry pi?
If you have node 4.0.0, are custom plugins with dependencies working in your case?
Thanks!

**Comment by digiltd** [comment]
What model Pi and what version of Raspbian are you running on it?

Also what versions of nodejs and npm are installed (type `nodejs -v` and `npm -v` to find out)

The #103 issue was merged with the wiki a while back and is safe to assume is outdated.

What issue are you having?

I'm running node 4.4.4 and npm 3.7.2 on my Pi 2.

**Comment by Ricardo1980** [comment]
Thanks for reply.

I bought a Raspberry Pi 3 a few days ago. I installed Raspbian and it is up-to-date.

Now my node version is 0.12, but yesterday was 4.4.4.

The problem I have is that the plugin I developed in my mac does not work in the RasPi, because the dependencies cannot be loaded.
Does custom plugins with dependencies work in your RasPi?
More info here:
https://github.com/nfarina/homebridge/issues/657

Any help is appreciated. Thanks.
